SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -263,19 +268,17 @@ final class SwedbankPaySDKViewModel: NSObject {
     /// - parameter successCallback: called on success
     /// - parameter errorCallback: called on failure
     /// - returns: `OperationsList` on successCallback, `SwedbankPaySDK.Problem` on errorCallback
-    private func handleResponse(_ response: DataResponse<Any>, successCallback: Closure<OperationsList>? = nil, errorCallback: Closure<SwedbankPaySDK.Problem>? = nil) {
-        if let responseValue = response.result.value {
+    private func handleResponse(_ response: AFDataResponse<OperationsList>, successCallback: Closure<OperationsList>? = nil, errorCallback: Closure<SwedbankPaySDK.Problem>? = nil) {
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 181 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)